app/services/event.rb

```diff
@@ -0,0 +1,36 @@
+##
+# Represents events occurring within the application that we are interested in tracking.
```

Comment:
I don't mind so much documentation like this that adds some context to a library style service